server/test/uk/gov/ons/addressIndex/server/controllers/AddressControllerSpec.scala

```diff
@@ -111,6 +118,12 @@ class AddressControllerSpec  extends PlaySpec with Results with AddressIndexCann
       Future.successful(HybridAddresses(Seq.empty, 1.0f, 0))
 
     override def client(): ElasticClient = ElasticClient.local(Settings.builder().build())
+
+    override def logger: Logger = ???
```

Comment:
We already log queries
https://github.com/ONSdigital/address-index-api/blob/develop/server/app/uk/gov/ons/addressIndex/server/modules/AddressIndexRepository.scala#L80
(and the same for query by uprn)